test/specs/core.scale.tests.js

```diff
@@ -208,6 +208,43 @@ describe('Core.scale', function() {
 		});
 	});
 
+	it('should add right padding for long tick labels', function() {
+		var chart = window.acquireChart({
+			type: 'line',
+			data: {
+				labels: [
+					'This is a very long label',
+					'This is a very long label'
+				],
+				datasets: [{
+					data: [0, 1]
+				}]
+			},
+			options: {
+				scales: {
+					xAxes: [{
+						id: 'foo'
+					}],
+					yAxes: [{
+						display: false
+					}]
+				},
+				legend: {
+					display: false
+				}
+			}
+		}, {
+			canvas: {
+				height: 100,
+				width: 200
+			}
+		});
+
+		var scale = chart.scales.foo;
+		expect(scale.left).toBeGreaterThan(100);
+		expect(scale.right).toBeGreaterThan(190);
```

Comment:
Without #6021, the same size of left- and right-padding is added (half width of the label text). But the left padding is greater and the right-padding is less with #6021. That's why `toBeGreaterThan` is used for the both side.

Reply:
but `scale.right` should also be less than 200, for there to be any padding, right ?

Reply:
That's right. Do you think `expect(scale.right).toBeLessThan(200)` is enough?

Reply:
Not  sure. Maybe its silly to even test, because the label is right aligned and there is not much padding in right side. Maybe just change the test name instead not to be about right padding :)

Reply:
Oh, I meant it checks if the “correct” padding is added. To be clear, I will change the test name.

Reply:
Done